### ISSUE TITLE
Can't get Emulator to work

### ISSUE THREAD

**Issue by st4ckz** [NationalSecurityAgency/ghidra#5723]
**Describe the bug**
Whenever I try to use the emulator, after setting a breakpoint and press resume, nothing happens. If I try to single step I get a ghidra.pcode.exec.DecodePcodeExecutionException: Unknown disassembly error. All I can do is step backwards or step outside the emulator. I've been testing with Visual Studio C++ executables.

**To Reproduce**
1. Open the executable with the emulator
2. Right click at the entry (or anywhere it doesn't matter) and select emulate Program in new trace
3. Set a breakpoint somewhere further along the chain of execution
4. Press resume, keep pressing resume...nothing happens.
5. Press single step and get ghidra.pcode.exec.DecodePcodeExecutionException: Unknown disassembly error (PC=00004f02)

**Expected behavior**
For the emulator to run and stop at breakpoints.

**Screenshots**

**Attachments**
If applicable, please attach any files that caused problems or log files generated by the software.
Application log attached. Relevant executables will be No Sleep and Hello World
[application.log](https://github.com/NationalSecurityAgency/ghidra/files/12482007/application.log)

**Environment (please complete the following information):**
 - OS: Windows 11
 - JVM Version: Amazon.com Inc. 17.0.8.1 - have also tried Eclipse and Oracle
 - Ghidra Version: 10.3.3
 - Ghidra Origin: official GitHub distro

**Additional context**

**Comment by st4ckz** [comment]
Unfortunately I can't get the debugger to work properly. Every attempt to start it is met with  "The resulting target process has no mapping to the static image Hello World.exe - .ProgramDB. Intervention is required before static and dynamic addresses can be translated. Check the target's module list.". The modules window will not populate no matter what I try.

**Comment by d-millar** [comment]
@st4ckz Just to clarify, are you having problems with the Emulator or the Debugger proper?  Or did you figure it out?

**Comment by st4ckz** [comment]
I worked out a way to get things working, though I initially kind of put the cart before the horse. I was having trouble with both the emulator and debugger. I redirected my efforts towards the debugger. I found that after selecting to debug an executable and receiving the "The resulting target process has no mapping to the static image" I then had to go the modules window, right click on the module and select "Map module to example.exe". Then in the resulting Map Modules window I had to click on the "Memorize" box for the module. After that I could successfully debug and emulate.
